OsmAnd/src/net/osmand/plus/OsmandPlugin.java

```diff
@@ -116,6 +117,7 @@ public static void initPlugins(OsmandApplication app) {
 		allPlugins.add(new AccessibilityPlugin(app));
 		allPlugins.add(new OsmEditingPlugin(app));
 		allPlugins.add(new OsmandDevelopmentPlugin(app));
+		allPlugins.add(new SmartNaviWatchPlugin(app));
```

Comment:
here it could be if(Android.SDK_BUILD >= 19) {
static function to create that class (and specify full classpath to the class without imports)
}